### PR TITLE
Use GCLOUD_PROJECT instead of GCLOUD_PROJECT_NUM

### LINKED ISSUE #236: Use GCLOUD_PROJECT instead of GCLOUD_PROJECT_NUM
This aligns with the environment variable that [gcloud](http://npm.im/gcloud) module is already using.

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #236

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

**Comment by ofrobots** [comment]
Looking at https://www.npmjs.com/package/gcloud, they are using `GCLOUD_PROJECT_ID`

**Comment by matthewloring** [comment]
Fixed.

**Comment by ofrobots** [comment]
For posterity, that was a bug in `gcloud`'s README file. The correct env. var. to use is GCLOUD_PROJECT.

File: README.md [hunk 1]
```diff
@@ -24,17 +24,17 @@ This module provides Cloud Trace support for Node.js applications. [Google Cloud
 
         npm install --save @google/cloud-trace
 
-2. Include and start the library at the *top of the main script of your application*. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
+2. If you are running locally, or not on the Google Cloud Platform, set the GCLOUD_PROJECT environment variable. Your project ID is visible in the [Google Cloud Console Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `particular-future-12345`. If your application is [running on Google Cloud Platform](running-on-google-cloud-platform), you don't need to specify the project ID.
```
Comment (ofrobots) [inline README.md]: I would put this at the end of this numbered list. Also, some rewording would help, something like...

"If your application is running outside of Google Cloud Platform, such as locally, on-premise, or on another cloud provider, you will need additional configuration. See the running elsewhere section."

File: README.md [hunk 2]
```diff
@@ -24,17 +24,17 @@ This module provides Cloud Trace support for Node.js applications. [Google Cloud
...
 
-        require('@google/cloud-trace').start({projectId: 'your-project-id'});
+3. Include and start the library at the *top of the main script of your application*. If you use `--require` in your start up command, make sure that the trace agent is --required first. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
```
Comment (ofrobots) [inline README.md]: Rewording..

"1. Require and start the trace module as the **very first thing** in your application... 
..require syntax..

It is imperative that the trace module is the very first thing that runs. See section XX for more details. *Add section explaining the --require issue.*.

File: README.md (hunk 1 above)
Comment (matthewloring) [inline README.md]: I think the quick start section should precisely describe what is required to run the trace agent the first time it is installed (which will likely not be on GCP). Looping in @justinbeckwith for insight on what would be most helpful here.

Comment (ofrobots) [inline README.md]: Then you also need to provide instructions on how to get authentication token in place.

Comment (JustinBeckwith) [inline README.md]: In the quick start instructions, for 2, I would just ask them to give us the project id - no hemming and hawing :)  Something like:

Set the GCLOUD_PROJECT environment variable. You can find your Project ID in the Google Cloud Developers Console, or by running the command `gcloud projects list`.  You can ensure this environment variable is set at startup time by placing it in your startup script in `package.json`:

```json
"scripts": {
  "start": "GCLOUD_PROJECT=<YOUR_PROJECT_ID> node server.js",
},
```

Then, further down in the readme, include a specific section (if there isn't one already) on running this outside of Google Cloud.

File: README.md (hunk 2 above)
Comment (JustinBeckwith) [inline README.md]: If you're feeling really nice, show an example.  If you're feeling super nice, warn people that it's `bin/www` in the case of an express app :)

**Comment by JustinBeckwith** [comment]
This is looking a lot better folks.  LGTM.  Lets get the GCLOUD_PROJECT change in, and then if needed we can bikeshed the readme prose.

File: README.md (hunk 2 above)
Comment (matthewloring) [inline README.md]: Sorry, I do not understand. What is `bin/www`?

**Comment by matthewloring** [comment]
@ofrobots @JustinBeckwith I've taken another editing pass at this.

File: README.md [hunk 3]
```diff
@@ -24,17 +24,24 @@ This module provides StackDriver Trace support for Node.js applications. [StackD
 
         npm install --save @google/cloud-trace
 
-2. Include and start the library at the *top of the main script of your application*. It's important that the trace agent is the first thing executed so that it can accurately gather data:
+2. Set the GCLOUD_PROJECT environment variable. You can find your Project ID in the [Google Cloud Developers Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), or by running the command `gcloud projects list`. You can ensure this environment variable is set at startup time by placing it in your startup script in `package.json`:
```
Comment (ofrobots) [inline README.md]: Can you use the out-of-line markdown link syntax and put the link at th bottom; and drop the `?_ga=...`

**Comment by ofrobots** [comment]
LGTM w/ nit, once the CI is green.

File: README.md (hunk 3 above)
Comment (matthewloring) [inline README.md]: Done.

**Pull request merged**